### ISSUE TITLE
i get nlp in folder structure instead of nlu , so i can't botonic train 

### ISSUE THREAD

**Issue by nadabouslama** [hubtype/botonic#2475]
the folder structure that i get after i create a new botonic prpject is different from the one indicating in the user guide , i don't have nlu folder it's files such as utterances , those differences are preventing me from training my bot

**Comment by Datcomx2** [comment]
Just wanted to bring this issue to light because the files are not updated for the NLU assistant model. When do you think that will be done by? Trying to implement that myself to my own chatbot but kinda hard to follow since the tutorial/files are not updated.

**Comment by nadabouslama** [comment]
Hello , i've actually faced a lot of issues with botonic , the documents
are not matching with actual program folder structure or code. so i
decided to not work with it.
